src/common/http/connection.cpp

```diff
@@ -0,0 +1,10 @@
+#include "connection.hpp"
+
+namespace http {
+
+Connection::Connection(uint8_t timeout_s)
+    : timeout_s(timeout_s) {}
+
+uint8_t Connection::get_timeout_s() const { return timeout_s; }
```

Comment:
Considering how trivial both these are, does it make sense to have them inline in the class?

Reply:
I don't know if we have a consensus or a rule. I use it based on how I feel at the time and nobody seemed to try beating mi with an existing rule or anything, so :shrug: